tsconfig.json

```diff
@@ -1,14 +1,15 @@
 {
   "compilerOptions": {
     "target": "ES6",
```

Comment:
Why is the target so conservative. Use at least ES2017 these days. 🙂

Reply:
Was an auto-gen config option, need to come back and update this.